connect/transforms/src/test/java/org/apache/kafka/connect/transforms/ExtractFieldTest.java

```diff
@@ -186,7 +188,94 @@ public void nonExistentNestedFieldWithSchemaShouldFail() {
 
     @Test
     public void testExtractFieldVersionRetrievedFromAppInfoParser() {
-        assertEquals(AppInfoParser.getVersion(), xform.version());
+        assertEquals(AppInfoParser.getVersion(), xformKey.version());
+    }
+
+    @Test
+    public void testUnsetOptionalFieldIsNull() {
```

Comment:
Is there a reason why you used `ParameterizedTest` in some places and multiple tests in others?